4_gap_analysis/src/3_conusMap.R

```diff
@@ -0,0 +1,75 @@
+#setwd("D:/abock/NAWQA/WQR/nawqa_wqp")
+
+# Bugs in tidyverse interfere with ggplot2 maps functionality
+# https://stackoverflow.com/questions/45066628/cannot-run-map-datastate
+# library(tidyverse)
+library(leaflet)
+library(tidyverse)
+library(mapview)
+library(htmlwidgets)
+library(webshot)
+
+#sapply(list.files("4_gap_analysis/src",full.names = T),source,.GlobalEnv)
+siteFiles<-dir('4_gap_analysis/in/Site', pattern='*sites.feather',full.names = T)
+# Files generated by filtering (by site)
+sampFiles<-dir('4_gap_analysis/in/Tiers', pattern='*.feather',full.names = T)
+
+# combined data may not work
+siteInfo <- bind_rows(
+  lapply(siteFiles, function(dfile) {
+    dat <- feather::read_feather(dfile)
+  })
+)
+siteInfo<-siteInfo %>% distinct(MonitoringLocationIdentifier,.keep_all=T)
+
+sampleInfo<- bind_rows(
+  lapply(sampFiles, function(dfile) {
+    dat <- feather::read_feather(dfile)
+  })
+)
+
+TierA<-sampleInfo %>%
+  filter(numYears > 4 & numCon==3 & Quart > 19) %>%
```

Comment:
Are numCon and Quart still current column names? I can see where NumConstituentsPerSample and NumQtrs1 get created but can't find where these come from. Might just be because I'm not seeing it, but figured I'd call it out in case you can check and decide whether this needs updating.

Reply:
This filtering code is removed on the current version I am now working off of; I thought it would be cleaner to have the mapping script with just mapping and visualization capabilities.